src/components/PeopleTable.tsx

```diff
@@ -1,5 +1,38 @@
+import React from 'react';
+import { Person } from '../types';
+import { Link, useParams, useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import PersonLink from './PersonLink';
+import { SearchLink } from './SearchLink';
+
+type Props = {
+  sortedPeople: Person[];
+  getSortIcon: (sorted: string) => string;
+  peoples: Person[];
+};
+
 /* eslint-disable jsx-a11y/control-has-associated-label */
-export const PeopleTable = () => {
+export const PeopleTable: React.FC<Props> = ({
+  sortedPeople,
+  getSortIcon,
+  peoples,
+}) => {
+  const [searchParams] = useSearchParams();
+  const { peopleId } = useParams();
+
+  const sort = searchParams.get('sort');
+  const reversed = searchParams.get('reversed');
+
+  const clikButtonsSort = (sorted: string) => {
+    if (sort === sorted && reversed) {
+      return { sort: null, reversed: null };
+    } else if (sort === sorted) {
+      return { sort: sorted, reversed: 'true' };
+    } else {
+      return { sort: sorted, reversed: null };
+    }
```

Comment:
```suggestion
    if (sort === sorted && reversed) {
      return { sort: null, reversed: null };
    }
    
      if (sort === sorted) {
      return { sort: sorted, reversed: 'true' };
    } 
    
      return { sort: sorted, reversed: null };
   
```
actually else is redundant here